redaxo/src/addons/install/lib/webservice.php

```diff
@@ -174,6 +228,12 @@ private static function loadCache()
         }
     }
 
+    /**
+     * Writess the given data into the local cache
+     *
+     * @param string $path
+     * @param mixed $data
```

Comment:
```suggestion
     * @param array $data
```

Glaube hier sind nur Arrays vorgesehen.